py3dtiles/points/node.py

```diff
@@ -208,7 +208,7 @@ def to_tileset(executor, name, parent_aabb, parent_spacing, folder, scale):
                     0, 0, half_size[2]]
             },
             # geometricError is in meter so cancel scale
-            'geometricError': 10 * node.spacing / scale[0],
+            'geometricError': 20 * node.spacing / scale[0],
```

Comment:
Let's not mix unrelated commits in a single PR.

I'm fine with changing the `geometricError` value, but you'll need to show that your proposed value works better on a couple of models.

Reply:
Sorry, this is not a good commit, 
I think we can provide a parameter for easy modification